source/docs/contributing/wpilib/index.rst

```diff
@@ -7,6 +7,16 @@ Developing with allwpilib
 
 This is a list of links to the various documentation for the `allwpilib <https://github.com/wpilibsuite/allwpilib>`__ repository.
 
+Quick Start
```

Comment:
Something like Quick Start Building might be more descriptive

Reply:
It's not really true, since building is only one step of 4.